packages/perseus/src/widgets/interactive-graphs/grid.tsx

```diff
@@ -0,0 +1,33 @@
+import {Coordinates} from "mafs";
+import * as React from "react";
+
+import type {SizeClass} from "../../util/sizing-utils";
+
+interface GridProps {
+    step: [number, number];
+    gridStep: [number, number];
+    range: [[number, number], [number, number]];
+    containerSizeClass: SizeClass;
+    markings: "graph" | "grid" | "none";
```

Comment:
I've seen this `markings` type in a couple places in `perseus-types.ts`. Should we have a single type for `markings` that we can then reference in each of these places (including here)?